src/lib.rs

```diff
@@ -54,6 +55,8 @@ where
     orientation: Orientation,
 }
 
+pub type DisplayNoRST<DI, MODEL> = Display<DI, no_pin::NoPin, MODEL>;
```

Comment:
Do we need another type tho? I'm thinking making `rst: Option<RST>` with the additional `no_rst` contstructor variants will be less confusing.

Reply:
It'll also force model implementors to account for this,  which atm. the ST7789 doesn't for example (if it works it works by accident. I think I should do a soft reset in that case).

Reply:
Do you mean the `NoPin` type or the `DisplayNoRST` type ?
1. I am using a custom NoPin type to avoid users to choose a random Pin when they mean to pass a None value for the RST Pin.
2. I am defining a DisplayNoRST type to avoid leaking the `NoPin` type externally

I don't know how it would be possible to use Option<RST> without needing to provide a specific Pin as a driver user, but any suggestion is welcome.

Reply:
I mean 2.

Leaking `NoPin` is inevitable in this case, but at least the user won't really care, they just do `Display::st7789_no_rst(di)` or such and that's it from their perspective.

Reply:
I tried to rewrite according to your remarks.

However you will see in the commit 48ed7cababb5d2da2a08ca4f7a47dea5aa6cf9ae that I took the init sequence from your old driver, as it solved the problem with the inverted colors for my TTGO T-Watch-2020.